Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/Validation/EconDamperPositionLimitsMultiZone_LoopDisable.mo

```diff
@@ -0,0 +1,183 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.AtomicSequences.Validation;
+model EconDamperPositionLimitsMultiZone_LoopDisable
+  "Validation model for the multiple zone VAV AHU minimum outdoor air control - damper position limits"
+  extends Modelica.Icons.Example;
+
+  parameter Types.FreezeProtectionStage freProStage1 = Types.FreezeProtectionStage.stage1
+    "Freeze protection stage 1";
+  parameter Types.FreezeProtectionStage freProStage2 = Types.FreezeProtectionStage.stage2
+    "Freeze protection stage 2";
+  parameter Integer freProStage1Num = Integer(freProStage1)-1
+    "Numerical value for freeze protection stage 1";
+  parameter Integer freProStage2Num = Integer(freProStage2)-1
+    "Numerical value for freeze protection stage 2";
+  parameter Types.OperationMode occupied = Types.OperationMode.occupied
+    "AHU operation mode is \"Occupied\"";
+  parameter Types.OperationMode warmUp = Types.OperationMode.warmUp
+    "AHU operation mode is \"Warmup\"";
+  parameter Integer occupiedNum = Integer(occupied)
+    "Numerical value for AHU operation mode \"Occupied\"";
+  parameter Integer warmUpNum = Integer(warmUp)
+    "Numerical value for AHU operation mode \"WarmUp\"";
+  parameter Real airflowSetpoint(unit="m3/s")=0.71
+    "Example volumetric airflow setpoint, 15cfm/occupant, 100 occupants";
+  parameter Real minSenOutVolAirflow(unit="m3/s")=0.61
+    "Volumetric airflow sensor output, minimum value in the example";
+  parameter Real senOutVolAirIncrease(unit="m3/s")=0.2
+    "Maximum increase in airflow volume during the example simulation";
+
+  // Fan Status
+  CDL.Logical.Constant FanStatus(k=false) "Fan is off"
```

Comment:
`FanStatus` is an instance and hence should be lower case, e.g., use `fanStatus`